bot/cogs/snakes.py

```diff
@@ -29,16 +29,19 @@
 EMPTY = u'\u200b'
 
 # Results
-TICK_EMOJI = "\u2705"  # Correct Peg, Correct Hole
+TICK_EMOJI = "\u2705"  # Correct peg, correct hole
 CROSS_EMOJI = "\u274C"  # Wrong
-BLANK_EMOJI = "\u26AA"  # Correct Peg Wrong Hole
+BLANK_EMOJI = "\u26AA"  # Correct peg, wrong hle
 
 # Holes
 HOLE_EMOJI = "\u2B1C"
 
 ANTIDOTE_EMOJI = [FIRST_EMOJI, SECOND_EMOJI, THIRD_EMOJI, FOURTH_EMOJI, FIFTH_EMOJI]
 
 
+def to_lower(argument):
```

Comment:
I don't see why this helper function is useful. it's also error prone. if you pass anything into it that isn't a string, it'll crash the program. Just use the `str.lower()` method instead of having this helper function at all.